NEWS

```diff
@@ -2,6 +2,15 @@ systemd System and Service Manager
 
 CHANGES WITH 257 in spe:
 
+        Incompatible changes:
+
+        * The --purge switch of systemd-tmpfiles (which was added in v256) has
+          been reworked: it will now only apply to tmpfiles.d/ lines marked
+          with the new "$" flag. This is an incompatible change, and means any
+          tmpfiles.d/ files which shall be used together with --purge need to
+          be updated accordingly. This change has been made to make it harder
+          to accidentally delete too many files when using --purge incorrectly.
```

Comment:
Ugh, all other incompatible changes are listed in the section below already

Reply:
this one is a compat break that now happened, the other ones are plans for the future

Reply:
actually there are some that are not for the future in the list below too, but we can rearrange later

Reply:
ah, indeed, missed those. yeah, we should rearrange this to have two sections, one for stuff we broke and one of stuff we intend to break.

Reply:
Ideally this can just be done in this PR?

Reply:
i started to rework NEWS in a bigger fashion in another git branch. would prefer to do that separately.